### PR TITLE
[Bug] Fix broken consensus tests on main branch - mocking StoreBlock in persistence module

### PR DESCRIPTION
**Pull request by Olshansk**
## Description

Fix a small bug on main introduced by removal of `UtilityModule`'s `StoreBlock` function.

## Issue

No associated issue. This is a quick followup to #254.

Screen Shot 2022-09-29 at 4 32 23 PM

## Type of change

Please mark the relevant option(s):

- [ ] New feature, functionality or library
- [x] Bug fix
- [ ] Code health or cleanup
- [ ] Major breaking change
- [ ] Documentation
- [ ] Other 

## List of changes

- Fixed broken test on main by mocking the `StoreBlock` function on the persistence module in a consensus test
- Remove the unused `StoreBlock` function in the utility module

## Testing

- [x] `make develop_test`
- [ ] [LocalNet](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md) w/ all of the steps outlined in the `README`

## Required Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested my changes using the available tooling
- [x] I have updated the corresponding CHANGELOG

### If Applicable Checklist
- [x] I have updated the corresponding README(s); local and/or global
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added, or updated, [mermaid.js](https://mermaid-js.github.io) diagrams in the corresponding README(s)
- [ ] I have added, or updated, documentation and [mermaid.js](https://mermaid-js.github.io) diagrams in `shared/docs/*` if I updated `shared/*`README(s)

### CODE REVIEW

**Comment by andrewnguyen22** [comment]
Finish PR description before requesting review

File: utility/doc/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.0.4] - 2022-09-29
```
Comment (andrewnguyen22) [inline utility/doc/CHANGELOG.md:10]: Versioning OB1?

**Review by andrewnguyen22 — CHANGES_REQUESTED** [review]

File: utility/doc/CHANGELOG.md (hunk 1 above)
Comment (andrewnguyen22) [inline utility/doc/CHANGELOG.md:10]: Versioning OB1?

**Review by andrewnguyen22 — CHANGES_REQUESTED** [review]
Versioning OB1

**Comment by Olshansk** [comment]
@andrewnguyen22 

> Finish PR description before requesting review

Sounds good. I jumped the 🔫 on this one.

**Comment by Olshansk** [comment]
Versioning...

giphy

**Comment by Olshansk** [comment]
Please note that `make develop_test` works but there is still an issue with LocalNet we're going to need to fix.

Screen Shot 2022-09-29 at 5 51 19 PM

**Review by andrewnguyen22 — APPROVED** [review]
@Olshansk LGTM, I realized you asked me to fix `actor_test` before merging and I did not see that until after I had merged. I planned on following up with another PR, but was wondering if you'd be able to patch that up in this PR. 

If not NBD -lmk

**Comment by Olshansk** [comment]
@andrewnguyen22 Patched, updated, pushed, verified & merged.

**Comment by Olshansk** [comment]
Also, I just misunderstood what you meant by `I realized you asked me to fix actor_test before merging`. Oddly enough `make develop_test` passed for me, so:

1. Can you submit a PR with those unit test fixes.
2. I will investigate why `make develop_test` passes

**Comment by andrewnguyen22** [comment]
@Olshansk can't reproduce the issue - seems fine

**Comment by Olshansk** [comment]
@andrewnguyen22 Following up here for posterity, fix is in #282

**Pull request merged**